src/random_test.cpp

```diff
@@ -107,8 +108,12 @@ int sum_of_all_options(Thd1 *thd) {
   }
 
   /* for 5.7 disable tablespace rename */
-  if (db_branch().compare("5.7") == 0)
+  if (db_branch().compare("5.7") == 0) {
     opt_int_set(ALTER_TABLESPACE_RENAME, 0);
+    opt_int_set(CREATE_UNDO_TABLESPACE, 0);
```

Comment:
you need to  add one more option undo_tablespaces
also, rather having three options,
I would suggest to have one option undo_tbs_sqls
based on a probability it either do active/inactive/drop/create.

Reply:
Fixed